include/aspect/particle/property/elastic_tensor_decomposition.h

```diff
@@ -109,7 +109,7 @@ namespace aspect
          * The tensors below can be used to project matrices to different symmetries.
          * See Browaeys and Chevrot, 2004.
          */
-        static const SymmetricTensor<2,21> projection_matrix_triclinic_to_monoclinic(
+        const SymmetricTensor<2,21> projection_matrix_triclinic_to_monoclinic(
```

Comment:
I should add that I wished we could make these variables `constexpr`, but `SymmetricTensor` is not a `constexpr` class.